components/InitializeSdk.js

```diff
@@ -1,16 +1,16 @@
 import React, { useState, useEffect } from "react";
 import BluedotPointSdk from "bluedot-react-native";
-import { useHistory } from "react-router";
+import { useNavigate } from "react-router";
 import { Button, Text, TextInput, View } from "react-native";
 import { sendLocalNotification } from "../helpers/notifications";
 import styles from "../styles";
 
-const PROJECTID = "<YOUR_PROJECT_ID_GOES_HERE>";
+const PROJECTID = "YOUR_PROJECT_ID_GOES_HERE";
```

Comment:
Interestingly the extra "<", ">" in the dummy value `<YOUR_PROJECT_ID_GOES_HERE>`  causes the iOS PointSDK to crash. Not a good impression if user launches the Min app and tap INITIALIZE right away. 
Will probably need to update other Min apps if they are using this same dummy value, or/and add handling in iOS SDK to cater for this special character case.